src/test/computer.test.js

```diff
@@ -0,0 +1,175 @@
+const Computer = require('../computer');
+const Ship = require('../ship');
+const GameBoard = require('../gameBoard');
+const Player = require('../player');
```

Comment:
We need to use ES6, export, import, syntax, as stated in the [Pathwright to this project](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622118/)
![image](https://user-images.githubusercontent.com/46115725/73208810-b2005280-410c-11ea-8801-b9168287bdcf.png).
- Instead of using require, we need to import this objects.
![image](https://user-images.githubusercontent.com/46115725/73209414-cee95580-410d-11ea-86a6-3dd05dd87a01.png)